lib/web_ui/lib/src/engine/text/paragraph.dart

```diff
@@ -255,7 +255,16 @@ class EngineParagraph implements ui.Paragraph {
       return;
     }
 
+    Stopwatch stopwatch;
+    if (Profiler.isBenchmarkMode) {
+      stopwatch = Stopwatch()..start();
+    }
     _measurementResult = _measurementService.measure(this, constraints);
+    if (Profiler.isBenchmarkMode) {
+      Profiler.instance.benchmark('text_layout', stopwatch.elapsedMicroseconds);
+      stopwatch.stop();
```

Comment:
For better accuracy I think you want to stop before sending the number.

Reply:
I think `stopwatch.elapsedMicroseconds` is being read before calling `Profiler.instance.benchmark()`. Nevertheless, I'll move the stop to be above :)

Reply:
Ah, you're right. I think you can just not call `stop` at all, although somehow seeing the call to `stop` gives me more confidence about what's being measured :)